packages/common/container-definitions/package.json

```diff
@@ -80,6 +80,9 @@
       "RemovedEnumDeclaration_BindState": {
         "forwardCompat": false,
         "backCompat": false
+      },
+      "InterfaceDeclaration_IContainerContext": {
+        "forwardCompat": false
```

Comment:
@vladsud  - I don't think we can make any changes like this (breaking so-called forwardCompat or backCompat) for IContainerContext in main.  In this case, an old implementation does not reflect the new interface. I know you add a runtime check, but this is actually causing a type compatibility problem in Bohemia codebase. I think we can work around it but just bringing to your attention.

Reply:
I think you are right. I assumed that interface would be used only internally, but it's clearly wrong assumption.